lib/old_archive_test.go

```diff
@@ -255,20 +257,13 @@ func TestDerivedExecutionDiscarding(t *testing.T) {
 			metadata: `blah`,
 			expError: "invalid character",
 		},
-		{
-			metadata: `{
-				"filename": "/test.js", "pwd": "/",
-				"k6version": "0.24.0"
-			}`,
-			expError: "missing options key",
-		},
```

Comment:
is this now broken?

Reply:
not quite, rather the clean up function was a bit overzealous and required the presence of `options` in the archive metadata... which, while not completely wrong by itself (since it will always be present right now), is not strictly required... post-1007, `k6 cloud script.js` would work even if there aren't any options

Reply:
hm ... okay, that seems fine